### ISSUE TITLE
[BUG] Running awscliv2 s3 cp | AttributeError: 'bool' object has no attribute 'returncode'

### ISSUE THREAD

**Issue by sheetalbhalgat** [youtype/awscliv2#15]
**Describe the bug**
When running awscliv2 s3 cp <source> <dest> --recursive
gettign below error
`awsv2 s3 cp $CI_PROJECT_DIR/viewer-dev/ s3://$VIEWER_S3_BUCKET/scripts/ --recursive`

```js
  File "/usr/local/bin/awsv2", line 8, in <module>
    sys.exit(main_cli())
  File "/usr/local/lib/python3.10/site-packages/awscliv2/main.py", line 59, in main_cli
    main(sys.argv[1:])
  File "/usr/local/lib/python3.10/site-packages/awscliv2/main.py", line 51, in main
    sys.exit(runner.run_awscli_v2_detached(namespace.other))
  File "/usr/local/lib/python3.10/site-packages/awscliv2/awscli_runner.py", line [95](https://gitlab.com/exfo/products/tandm/basecamp/reporting/viewers/olx/-/jobs/2148327738#L95), in run_awscli_v2_detached
    return self._run_detached_subprocess(cmd)
  File "/usr/local/lib/python3.10/site-packages/awscliv2/awscli_runner.py", line 71, in _run_detached_subprocess
    return process.returncode
AttributeError: 'bool' object has no attribute 'returncode'
```

**To Reproduce**

```js
`FROM python:3.8-slim

RUN pip3 install requests awscliv2==2.0.2
RUN awsv2 --install
RUN awsv2 --configure default <access_key> <secreat_key>
RUN awsv2 --configure set region <region>
RUN echo "README" > README.md
RUN awsv2 s3 cp README.md s3://tms-web-viewers-scripts/scripts/README.md`
```

3. Actual behavior
giving error 'AttributeError: 'bool' object has no attribute 'returncode''

**Expected behavior**
awscliv2 s3 cp should work

*Additional context**
tried with awscliv2==2.0.2 and it works. Seems issues with latest version
https://github.com/vemel/awscliv2/compare/2.0.2...2.1.0

**Comment by vemel** [comment]
Thank you, will fix it shortly.

**Comment by vemel** [comment]
I have just released `2.1.1`. Please try it out and let me know if it works as it should.

**Comment by sheetalbhalgat** [comment]
Thanks for providing solution quickly. It works!!
